autopush/websocket.py

```diff
@@ -986,6 +1000,9 @@ def finish_webpush_notifications(self, result):
             self.ps.updates_sent[str(notif.channel_id)].append(notif)
             msg = notif.websocket_format()
             messages_sent = True
+            update_cnt += 1
```

Comment:
Oh, sorry, this won't work. We only fetch 10 at a time, so the max number this will ever be here is 10.